api/AnimationEvent.json

```diff
@@ -0,0 +1,232 @@
+{
+  "api": {
+    "AnimationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null,
+            "prefix": "webkit"
+          },
+          "chrome": [
+            {
+              "version_added": "1",
```

Comment:
Please swap this around, the more relevant version is the unprefixed one. (the first element will be rendered as the main compat cell in the MDN tables, other version ranges will be put into the expandable details.)

```json
        "chrome": [
            {
              "version_added": "43"
            }
            {
              "version_added": "1",
              "prefix": "webkit"
            },
          ],
```